### DISCUSSION TITLE
snponly.efi with console & colour command support

### DISCUSSION THREAD

**Discussion by schorschii** [ipxe/ipxe#673] [Q&A] [answered]
I'm currently using `snponly.efi` to boot Fujitsu server (normal `ipxe.efi` stuck in "iPXE initializing devices..." on these devices) but I don't wanna miss the `console` and `colour` commands. How can I compile a `snponly.efi` with those features?

**Comment by schorschii** [comment, marked as answer]
Sorry for the question, I was too hasty. It works just like compiling the regular `ipxe.efi`. I just needed to enable `#define CONSOLE_CMD` in `config/general.h`.

**Reply by NiKiZe** [reply]
Exactly so, and mentioned in documentation https://ipxe.org/cmd/console#build_options

One suggestion, use config/local/general.h and friends to override specific settings, that way you don't have to care on git pulls.

**Comment by stappersg** [comment]
On Tue, Jul 05, 2022 at 05:25:58AM -0700, Georg wrote:
> Sorry for the question, I was too hasty.

Please take some time to mark this discussion as "answered".

**Reply by schorschii** [reply]
I already did that?!

**Reply by stappersg** [reply]
Today I did learn that github.com email notification does **not** tell about _answered status_.

@schorschii  [thanks](https://giveupgithub.org/)
